samples/HealthChecks.UIAndApiCustomization/Startup.cs

```diff
@@ -64,6 +64,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                     {
                         setup.UIPath = "/show-health-ui"; // this is ui path in your browser
                         setup.ApiPath = "/health-ui-api"; // the UI ( spa app )  use this path to get information from the store ( this is NOT the healthz path, is internal ui api )
+                        setup.PageTitle = "Health Checks UI"; // the page title in <head>

```

Comment:
```suggestion
                        setup.PageTitle = "My wonderful Health Checks UI"; // the page title in <head>

```

Just to show the title differ from the standard one. OK?

Reply:
Done.